tests/bearlib/languages/documentation/DocBaseClassTest.py

```diff
@@ -254,5 +255,25 @@ def test_extract_documentation_PYTHON3_4(self):
         # Nothing is yielded as triple quote string literals are being
         # ignored.
         self.assertEqual(
-            list(extract_documentation(data, 'PYTHON3', 'default')),
+            list(DocBaseClass.extract(data, 'PYTHON3', 'default')),
             [])
+
+    def test_diff_generation(self):
```

Comment:
Check the bear part... we only use `generate_diff` if `doc_comment != new_comment` .still we need a case for that ?